Tests/test_Muscle_tool.py

```diff
@@ -132,14 +142,22 @@ def test_Muscle_profile_simple(self):
     def test_Muscle_profile_with_options(self):
         """Profile alignment, and switch and valued options."""
         # Using some keyword arguments, note -stable isn't supported in v3.8
-        cmdline = MuscleCommandline(muscle_exe, out=self.outfile4,
-                                    in1=self.infile2, in2=self.infile3,
-                                    profile=True, stable=True,
-                                    cluster1="neighborjoining")
-        self.assertEqual(str(cmdline), _escape_filename(muscle_exe) +
-                         " -out Fasta/temp_align_out4.fa" +
-                         " -profile -in1 Fasta/fa01 -in2 Fasta/f001" +
-                         " -cluster1 neighborjoining -stable")
+        cmdline = MuscleCommandline(
+            muscle_exe,
+            out=self.outfile4,
+            in1=self.infile2,
+            in2=self.infile3,
+            profile=True,
+            stable=True,
+            cluster1="neighborjoining",
+        )
+        self.assertEqual(
+            str(cmdline),
+            _escape_filename(muscle_exe)
+            + " -out Fasta/temp_align_out4.fa"
+            + " -profile -in1 Fasta/fa01 -in2 Fasta/f001"
+            + " -cluster1 neighborjoining -stable",
+        )
```

Comment:
Again, can you merge those strings into one or two lines line?

Reply:
Done.